scipy/special/add_newdocs.py

```diff
@@ -1628,23 +1628,111 @@ def add_newdoc(name, doc):
 
 add_newdoc("cosdg",
     """
-    cosdg(x)
+    cosdg(x, out=None)
 
     Cosine of the angle `x` given in degrees.
+
+    Parameters
+    ----------
+    x : array_like
+        Angle, given in degrees.
+    out : ndarray, optional
+        Optional output array for the function results.
+
+    Returns
+    -------
+    scalar or ndarray
+        Cosine of the input.
+
+    See Also
+    --------
+    sindg, tandg, cotdg
+
+    Examples
+    --------
+    >>> import scipy.special as sc
+
+    It is more accurate than using cosine directly.
```

Comment:
`cosine` is ambiguous here, as SciPy doesn't have it's own `cosine` function. (Well it does have two `cosine` functions, in `scipy.signal.windows` and `scipy.spatial`, but neither of those is what you are referring to here, and `scipy.cos` is deprecated.)
Instead use `np.cos`.  E.g.

```
It is more accurate than using np.cos directly.
```
The same comment applies in a few places below --- I'll add a simplified comment at each location.